File: templates/matlab/hvdc_facts_scenario_facets.m
function fig = hvdc_facts_scenario_facets()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('small_multiples', 3709, 'HVDC and FACTS analysis: scenario small multiples', 'HVDC and FACTS analysis', 'scenario small multiples');
end

File: templates/python/hvdc_facts_scenario_facets.py
"""hvdc_facts_scenario_facets: HVDC 与 FACTS场景分面（small-multiples 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='HVDC and FACTS analysis: scenario small multiples'):
    return make_template_figure('small_multiples', seed=3709, title=title, domain='HVDC and FACTS analysis', topic='scenario small multiples')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
